src/components/themed/EdgeText.tsx

```diff
@@ -36,31 +44,65 @@ export const EdgeText = (props: Props) => {
   )
 }
 
-export const Paragraph = (props: { children: React.ReactNode; paddingRem?: number[] | number; isWarning?: boolean }) => {
-  const { children, isWarning, paddingRem } = props
+/** A properly spaced block of text with default font color and size. Children
+ * that are wrapped in other `___Text` component types will override those defaults.
+ *
+ * A `Paragraph` *can* have `marginRem`, but *only* to avoid an extra `View` for
+ * spacing out `Paragraph(s)` in relation to their parent, *NOT* to give special
+ * spacing *between* `Paragraphs` */
+export const Paragraph = (props: {
+  children: React.ReactNode
+
+  /** @deprecated A `Paragraph` *can* have `marginRem`, but *only* to avoid an extra `View` for spacing out `Paragraph(s)` in relation to their parent, *NOT* to give special spacing *between* `Paragraphs`. It's still preferable to have the parents deal with spacing outside of `Paragraphs`. */
+  marginRem?: number[] | number
+}) => {
+  const { children, marginRem } = props
   const theme = useTheme()
   const styles = getStyles(theme)
-  const padding = sidesToPadding(mapSides(fixSides(paddingRem, 0.5), theme.rem))
-  const warningStyle = isWarning ? styles.warningText : null
+  const margin = sidesToMargin(mapSides(fixSides(marginRem, 0.5), theme.rem))
 
   return (
-    <EdgeText numberOfLines={0} disableFontScaling style={[padding, warningStyle]}>
+    <Text style={[styles.common, margin, androidAdjustTextStyle(theme)]} numberOfLines={0} adjustsFontSizeToFit={false}>
       {children}
-    </EdgeText>
+    </Text>
   )
 }
 
+/** Small (0.75rem) text.
+ * Unless used within a `Paragraph` block, provides no outer spacing. */
```

Comment:
```ts
/**
 * Makes the contents of an `EdgeText` or `Paragraph` smaller (0.75rem).
 */
```

We should be clear that this is only for use in nested situations, not as a stand-alone text element.